pic-configure

```diff
@@ -73,20 +74,29 @@ get_backend_flags()
 }
 
 # options may be followed by one colon to indicate they have a required argument
-OPTS=`getopt -o i:a:c:p:t:h -l install:,arch:,cmake:,params:,help -- "$@"`
+OPTS=`getopt -o i:b:c:p:t:h -l install:,backend:,cmake:,params:,help -- "$@"`
 if [ $? != 0 ] ; then
     # something went wrong, getopt will put out an error message for us
     exit 1
 fi
 
 # configure a specific preset in the cmakeFlags file
+# note: can be overwritten with a command line flag
 cmakeFlagsNr=0
 
+# set a default backend (and architecture) if supplied via environment var
+# note: can be overwritten with a command line flag
+pic_backend=${PIC_BACKEND:-""}
```

Comment:
@psychocoderHPC I just realized that we could, if we want, default to `"cuda"` here.

Reply:
offline discussed: will be added in a follow up PR